modules/installation/pages/install-ibmz.adoc

```diff
@@ -0,0 +1,104 @@
+[[installation-zsystems]]
+= Installing on {ibmz}
+
+
+This section is intended for z/VM systems programmers responsible for operating the {ibmz} mainframes.
+It assumes that you are a z/VM systems programmer trained on {ibmz} operating protocols, and steps you through installing {productname} onto an existing mainframe system.
+This section does not cover the variety of hardware configuration profiles available on {ibmz}, but provides a foundational overview of the procedure and requirements necessary for a successful {productname} Server deployment on {ibmz}.
+
+
+
+== System Requirements
+
+Before you begin, check that your environment meets the base system requirements.
+
+The base system for {productname}{nbsp}{productnumber} is {slsa}{nbsp}{sles-version}{nbsp}{sp-version}.
+
+.Compatible {ibmz} Systems:
+
+* IBM zEnterprise System z196
+* IBM zEnterprise System z114
+* IBM zEnterprise EC12
+* IBM zEnterprise EC12
+* IBM zEnterprise BC12
+* IBM z13
+* LinuxOne Rockhopper
+* LinuxOne Emperor
+
+
+[cols="1,1", options="header"]
+.Hardware Requirements
+|===
+| Hardware     | Recommended
+| CPU          | Minimum 4 dedicated 64-bit CPU cores
+| RAM:         | Test Server: Minimum 3{nbsp}GB RAM and 2{nbsp}GB Swap space
+|              | Base Installation: Minimum 16{nbsp}GB
+|              | Production Server: Minimum 32{nbsp}GB
+| Disk Space:  | Root Partition: Minimum 100{nbsp}GB
+|              | [path]``/var/lib/pgsql``: Minimum 50{nbsp}GB
+|              | [path]``/var/spacewalk``: Minimum 50{nbsp}GB per SUSE product and 250{nbsp}GB per Red Hat product
+|===
+
+[IMPORTANT]
+====
+Memory should be split across available RAM, VDISK, and swap to suit your environment.
+On a production system the ratio of physical memory to VDISK will need to be evaluated based on the number of clients you will be installing.
+====
+
+You will require an additional disk for database storage.
+This should be an [systemitem]``zFCP`` or [systemitem]``DASD`` device as these are preferred for use with [systemitem]``HYPERPAV``.
+The database storage disk should have:
+
+* At least 50{nbsp}GB for [path]``/var/lib/pgsql``
+* At least 50{nbsp}GB for each SUSE product in [path]``/var/spacewalk``
+* At least 250{nbsp}GB for each Red Hat product in [path]``/var/spacewalk``
+
+You will need to ensure you have sufficient disk storage for {productname}
+before running [command]``yast2 susemanagersetup``.
+By default, the {productname} file system, including the embedded database and patch directories, reside within the root directory.
+While adjustments are possible once installation is complete, it is important that specify and monitor these adjustments closely.
+For information on storage management and reclaiming disk space, see the {productname} Administration Guide.
```

Comment:
is it possible to provide a clickable link?  Wondering whether that's actually in our Admin Guide.  Maybe, it's in the SLES Admin?

Reply:
Oh yeah, I meant to come back to that. I will need to investigate.

Reply:
I can't see anything specific in our Admin Guide, but I think it's a good candidate for a troubleshooting  section. I'll create something in a different PR.

Reply:
https://github.com/SUSE/doc-susemanager/pull/437 (it's a beginning)